kong/db/schema/metaschema.lua

```diff
@@ -141,7 +141,6 @@ local meta_errors = {
   FIELDS_KEY = "each key in fields must be a string",
   ENDPOINT_KEY = "value must be a field name",
   TTL_RESERVED = "ttl is a reserved field name when ttl is enabled",
-  TTL_CREATED_AT = "ttl can only be enabled on entities that have a 'created_at' timestamp field",
```

Comment:
Why are we removing this constraint?

Reply:
At first I thought I need that when I created PR for `ttl=true` for the schema. But when I implemented Postgres strategy, I kinda saw it as optional. PG strategy will still count TTL if there is `created_at` and on updates `updated_at`, but there is also a fallback for entities that have `ttl=true`, but no `created_at`, in which case PG ttl is counted like `ngx.time() + ttl`. Cassandra never really needed or used `created_at` for `ttl` as it just accepts the number as is. With PG there is no native support as of now, so we decided to store `expires_at` (absolute time) to `ttl` field, as it makes it easier to cleanup, and exclude expired rows.